fpdf/fpdf.py

```diff
@@ -1974,7 +1974,15 @@ def _out(self, s):
         if(self.state == 2):
             self.pages[self.page]["content"] += (s + "\n")
         else:
-            self.buffer += (s + "\n")
+            if PY3K:
+                if not isinstance(s, bytes):
```

Comment:
But in Py 3, the bytes cases should already be converted to Latin-1 str by the decode("latin1") case at line 1967.